### ISSUE TITLE
Deprecations: The TCA property ['ctrl']['cruser_id'] of table 'xxx'  is not evaluated anymore and has therefore been removed. Please adjust your TCA accordingly.

### ISSUE THREAD

**Issue by SSFGizmo** [FriendsOfTYPO3/extension_builder#744]
> The TCA property ['ctrl']['cruser_id'] of table 'xxx'  is not evaluated anymore and has therefore been removed. Please adjust your TCA accordingly.

OLD
```
/Resources/Private/CodeTemplates/Extbase/Configuration/TCA/tableName.phpt

        'crdate' => 'crdate',
        'cruser_id' => 'cruser_id',<f:if condition="{domainObject.sorting}">
        'sortby' => 'sorting',</f:if><f:if condition="{extension.supportVersioning}">

```

* https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.0/Breaking-98024-TCA-option-cruserid-removed.html

Perhaps it can help
NEW
```
/Resources/Private/CodeTemplates/Extbase/Configuration/TCA/Overrides/tt_content.phpt


        'crdate' => 'crdate',<f:if condition="{domainObject.sorting}">
        'sortby' => 'sorting',</f:if><f:if condition="{extension.supportVersioning}">
```

**TYPO3 Version**
v12.4

**Extension Builder Version:**
v12.0.0-beta.2

**Comment by PKuhlmay** [comment]
fixed by #747

**Comment by DavidBruchmann** [comment]
Thanks for reporting!
